packages/core/src/components/Card/index.tsx

```diff
@@ -8,17 +8,32 @@ export { Content };
 export type CardProps = {
   /** List of `Content`s blocks to contain content. */
   children: NonNullable<React.ReactNode>;
+  /** Hide shadow. */
+  noShadow?: boolean;
   /** Set overflow to be visible. */
   overflow?: boolean;
+  /** Whether the card is selected or not. */
+  selected?: boolean;
   /** Custom style sheet. */
   styleSheet?: StyleSheet;
 };
 
 /**
  * An abstract layout to use as a base for cards.
  */
-export default function Card({ children, overflow, styleSheet }: CardProps) {
+export default function Card({ children, noShadow, overflow, selected, styleSheet }: CardProps) {
   const [styles, cx] = useStyles(styleSheet ?? styleSheetCard);
 
-  return <div className={cx(styles.card, overflow && styles.card_overflow)}>{children}</div>;
+  return (
+    <div
+      className={cx(
+        !noShadow && styles.card,
```

Comment:
I think the way you've written this breaks the component. `styles.card` sets 
- `pattern.box` (`border/boxShadow/borderRadius`) 
- `backgroundColor`
- `overflow`

and your `styles.card_noShadow` only applies `border` and ignores the rest. it seems like we could **always** apply `styles.card`, and re-define `card_noShadow` to only override `boxShadow`?

Reply:
Good catch! :)